conftest.py

```diff
@@ -122,3 +123,36 @@ def pytest_configure(config):
     #
     # Somehow the default database is always in memory, though.
     settings.DATABASES['transient']['TEST_NAME'] = ':memory:'
+
+
+@pytest.yield_fixture()
+def installed_packages_catalog():
+    with mock.patch('ideascube.serveradmin.catalog.Catalog', spec=True) as MockClass:
+        pseudo_package1 = mock.NonCallableMock(spec_set='id name description template_id language staff_only'.split())
```

Comment:
I didn't know about `spec_set`, that's really cool!

(Line is too long though, if you had used a list instead of `split`ting a string you could have more easily wrapped it.)